virtwho/runner.py

```diff
@@ -424,21 +425,22 @@ def mappings_local(self, rhsm_log):
         """
         data = dict()
         key = "Domain info:"
-        rex = re.compile(r'(?<=Domain info: )\[.*?\]\n+(?=\d\d\d\d|$)', re.S)
-        mapping_info = rex.findall(rhsm_log)[0]
-        try:
-            mapping_info = json.loads(mapping_info.replace('\n', ''),
-                                      strict=False)
-        except:
-            logger.warning(f"json.loads failed: {mapping_info}")
-            return data
-        for item in mapping_info:
-            guestId = item['guestId']
-            attr = dict()
-            attr['state'] = item['state']
-            attr['active'] = item['attributes']['active']
-            attr['type'] = item['attributes']['virtWhoType']
-            data[guestId] = attr
+        if key in rhsm_log:
```

Comment:
for `local` mode, if failed to get the mapping, there will be no `Domain info` key word in the mapping, so the function will fail. now changed to return a null dict for this situation when call for the mapping data for local.